vignettes/corridor-segmentation.Rmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "3. Corridor segmentation"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{3. Corridor segmentation}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(CRiSp)
+```
+
+For a more detailed analysis of an urban river corridor, corridor-level delineation may not be sufficient. The corridor needs to be subdivided into smaller morphological units. Segmentation is a process of subdividing the corridor by using major transversal road or rail infrastructure lines.
+
+By default, the all-in-one function `delineate_corridor()` divides the corridor into segments. It is also possible to use the `delinate_segment()` function to divide the corridor in a separate step. To demonstrate this as a separate step, we will use the built-in `bucharest_corridor` and `bucharest_streets` as input.
```

Comment:
Maybe it would be good to link to the corridor-delineation vignette where `delineate_corridor()` is explained.

also a small textual suggestion:
```suggestion
By default, the all-in-one function `delineate_corridor()` includes the division of the corridor into segments. It is also possible to use the `delineate_segment()` function to divide the corridor in a separate step. 
To demonstrate the delineation of segments as a separate step in the following, we will use the built-in `bucharest_corridor` and `bucharest_streets` as input.
```